wear/src/main/java/io/homeassistant/companion/android/tiles/TemplateTile.kt

```diff
@@ -72,7 +72,7 @@ class TemplateTile : TileService() {
                     Timeline.Builder().addTimelineEntry(
                         TimelineEntry.Builder().setLayout(
                             Layout.Builder().setRoot(
-                                layout(renderedText)
+                                layout(renderedText.toString())
```

Comment:
This will result in the text "null" on the tile if rendering the template failed (because `integrationUseCase.renderTemplate` will return `null` on failure and `toString()` on that will give the text "null").
Lines 60-64 should probably be changed to also use the error string when the result is `null`.

Reply:
Done